src/openzeppelin/introspection/erc165/lib.cairo

```diff
@@ -0,0 +1,6 @@
+mod tests;
+
+trait IERC165 {
+  fn supports_interface(interface_id: felt) -> bool;
+  fn register_interface(interface_id: felt);
```

Comment:
This function is not part of the ERC. Should it be in the trait ? 

IMO, `_register_interface` should be an "internal" function to the module, and not exposed

Reply:
you thumbed up, but in the merge version, `register_interface` is still external.